src/replication.c

```diff
@@ -1827,72 +1827,93 @@ void readSyncBulkPayload(connection *conn) {
     return;
 }
 
-/* Send a synchronous command to the master. Used to send AUTH and
- * REPLCONF commands before starting the replication with SYNC.
+char* receiveSynchronousResponse(connection *conn) {
```

Comment:
nit: normally the * sticks with the function. Same with below. 
```suggestion
char *receiveSynchronousResponse(connection *conn) {
```